Google.PowerShell/BigQuery/BqTabledata.cs

```diff
@@ -0,0 +1,135 @@
+// Copyright 2015-2017 Google Inc. All Rights Reserved.
+// Licensed under the Apache License Version 2.0.
+
+using Google.Apis.Bigquery.v2;
+using Google.Apis.Bigquery.v2.Data;
+using Google.PowerShell.Common;
+using System;
+using System.Net;
+using System.Management.Automation;
+using System.Collections.Generic;
+
+namespace Google.PowerShell.BigQuery
+{
+    /// <summary>
+    /// <para type="synopsis">
+    /// Instantiates a new BQ schema or adds a field to a pre-existing schema.
+    /// </para>
+    /// <para type="description">
+    /// If no existing schema is passed in, this command will create a new schema with one column. 
+    /// If an existing schema is supplied, this command will add a new column to that schema. 
+    /// Required fields for each column are Name and Type. Possible values for Type include STRING, 
+    /// BYTES, INTEGER (also called INT64), FLOAT (also FLOAT64), BOOLEAN (also BOOL), TIMESTAMP, 
+    /// DATE, TIME, DATETIME, and RECORD (where RECORD indicates that the field contains a nested 
+    /// schema. Also called STRUCT). Possible values for Mode include NULLABLE, REQUIRED, and 
+    /// REPEATED. Case is ignored for both Type and Mode. This command returns the new or modified 
+    /// TableSchema object.
+    /// </para>
+    /// <example>
+    ///   <code>
+    /// PS C:\> $table = Get-BqDataset “book_data” | Get-BqTable "21st_century"
+    /// PS C:\> $schema = New-BqSchema -Name "Title" -Type "STRING" -Description "Book Title"
+    /// PS C:\> $schema = $schema | New-BqSchema -Name "Author" -Type "STRING" -Description "Book Author"
+    /// PS C:\> $table.Schema = $schema
+    /// PS C:\> $table | Set-BqTable
+    ///   </code>
+    ///   <para>This will create a new schema, assign it to a table, and then send the 
+    ///   revised table to the server to be saved.</para>
+    /// </example> 
+    /// <para type="link" uri="(https://cloud.google.com/bigquery/docs/reference/rest/v2/tables)">
+    /// [BigQuery Tables]
+    /// </para>
+    /// </summary>
+    [Cmdlet(VerbsCommon.New, "BqSchema")]
```

Comment:
I feel like this cmdlet should probably be named "Add-BqFieldSchema" instead of "New-BqSchema" (the -InputObject parameter can be named "BqSchema"). @chrsmith and @ILMTitan what do you guys think?